core/server/worker/src/test/java/com/google/common/util/concurrent/MockRateLimiter.java

```diff
@@ -86,7 +85,7 @@ private void sleepMillis(int millis) {
     }
 
     private void sleepMicros(String caption, long micros) {
-      mInstant += MICROSECONDS.toNanos(micros);
+      mInstant += micros;
```

Comment:
consider renaming `mInstant` to `mInstantMicros` to help avoid bugs like this in the future